### DISCUSSION TITLE
Create a file naming convention

### DISCUSSION THREAD

**Discussion by milkbump** [vlang/v#11673] [Ideas]
Looking through V's source files, the naming convention doesn't seem entirely consistent. While I've mostly seen `snake_case`, I've also seen `kebab-case`. It seems to be a [point of contention in Go](https://github.com/golang/go/issues/36060). So now would be a nice time to use and document a convention. 

Notable comment from the linked Go issue above:

> A short new section on CodeReviewComments is more than sufficient. 

Example:

> There is a de facto standard for V source file names: all lowercase with underscore separation when necessary, i.e. snake case.

Thoughts?

Edit: most of the snake case violations happen in v/cmd/tools/

**Comment by spytheman** [comment]
They are named like that, since they match the name of the command that they implement:
`v test-fixed` => `cmd/tools/vtest-fixed.v`
`v setup-freetype.v` => `cmd/tools/vsetup-freetype.v`
... and so on.

I guess it was just easier to type the name, without using shift at the time. It is easy to rename them, if we want to enforce a naming convention using _ instead of - for the file names.

@medvednikov what do you think?
